cfme-performance/utils/providers.py

```diff
@@ -410,6 +487,95 @@ def refresh_provider_vms_bulk(vm_ids):
         round(time.time() - starttime, 2)))
 
 
+def provision_vm(vm_name, template_guid):
+    """Create a provision request for a VM"""
+    starttime = time.time()
+    data_dict = {
+        'action': 'create',
+        'resources': [{
+            'template_fields': {
+                'guid': template_guid
+            },
+            'vm_fields': {
+                'number_of_sockets': 1,
+                'cores_per_socket': 1,
+                'vm_name': vm_name,
+                'vm_memory': '1024',
+                'vlan': 'VM Network',
+                'vm_auto_start': True,
+                'provision_type': 'native_clone'
+            },
+            'requester': {
+                'user_name': 'admin',
+                'owner_first_name': 'FirstName',
+                'owner_last_name': 'LastName',
+                'owner_email': 'alex@perf.com',
+                'auto_approve': True
+            },
+            'additional_values': {
+                'request_id': '1001'
+            },
+
+            'ems_custom_attributes': {},
+            'miq_custom_attributes': {}
+        }]
+    }
+
+    data_json = json.dumps(data_dict)
+    appliance = cfme_performance['appliance']['ip_address']
+    response = requests.post("https://" + appliance + "/api/provision_requests/",
+                             data=data_json,
+                             auth=(cfme_performance['appliance']['rest_api']['username'],
+                                   cfme_performance['appliance']['rest_api']['password']),
+                             verify=False,
+                             headers={"content-type": "application/json"},
+                             allow_redirects=False)
+
+    if response.status_code != 200:
+        logger.debug(response.text)
+    logger.debug('Provisioned VM {} in: {}s'.format(vm_name,
```

Comment:
Lets reword that to "queued" since the provisioning takes some time and we wouldn't want to get confused when reading the log output.